### PR TITLE
feat: Knowledge library page created

### PR DESCRIPTION
**Pull request by ejim11**
# Changes proposed

## What were you told to do?
Knowledge Library
All users should be able to access the library.

## What did you do?
I created the knowledge-library page which contains different categories of the library. THe library is yet to be populated though.

# Check List (Check all the applicable boxes)

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/ Videos

### CODE REVIEW

**Review by JulianaSau — APPROVED** [review]

**Comment by JulianaSau** [comment]
resolve conflicts @ejim11 so that i can merge

**Comment by JulianaSau** [comment]
And move this new page youve created to the static folder located here https://github.com/zurichat/zc_main/blob/dev/packages/main/src/pages/static

**Review by JulianaSau — APPROVED** [review]
Effect the changes requested above

**Review by JulianaSau — CHANGES_REQUESTED** [review]
Effect the requested changes above

File: packages/main/src/pages/KnowledgeLibrary/Categories/Categories.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import navData from "../Navigation/NavData";
+import classes from "./Categories.module.css";
+import { Link } from "react-router-dom";
+
+function Categories(props) {
+  return (
+    <div className={classes["container"]}>
+      <p className={`${classes["header-p"]} ${props.className}`}>Categories</p>
+      {navData.map(item => (
+        <div className={classes["item-box"]} key={item}>
```
Comment (ickynavigator) [inline packages/main/src/pages/KnowledgeLibrary/Categories/Categories.jsx:10]: don't use the whole object as a key

File: packages/main/src/pages/KnowledgeLibrary/Categories/Categories.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+          <p className={classes["title"]}>{item.title}</p>
+          <div className={classes["links-box"]}>
+            {item.subtitles.map((sub, i) => (
+              <Link key={i} to={sub.toLowerCase()}>
```
Comment (ickynavigator) [inline packages/main/src/pages/KnowledgeLibrary/Categories/Categories.jsx:14]: an index is not a key

File: packages/main/src/pages/KnowledgeLibrary/Categories/CategoryList.jsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import { useState } from "react";
+import classes from "./CategoryList.module.css";
+import rightLogo from "../assets/right.svg";
+import downLogo from "../assets/down.svg";
```
Comment (ickynavigator) [inline packages/main/src/pages/KnowledgeLibrary/Categories/CategoryList.jsx:4]: did you confirm that these icons are not available in react icons?

File: packages/main/src/pages/KnowledgeLibrary/Categories/CategoryList.module.css [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+.category ul li {
+  padding: 0.2rem 0;
+}
+
+.display {
+  display: block;
+  /*overflow: hidden;
+  */
```
Comment (ickynavigator) [inline packages/main/src/pages/KnowledgeLibrary/Categories/CategoryList.module.css:16]: remove commented styles

File: packages/main/src/pages/KnowledgeLibrary/Categories/CategoryList.module.css [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  height: min-content;
+  will-change: height;
+  transition: height 0.5s ease-in 0s;
+}
+
+.hide {
+  display: none;
+  /*overflow: hidden;
+  */
```
Comment (ickynavigator) [inline packages/main/src/pages/KnowledgeLibrary/Categories/CategoryList.module.css:25]: commented styles

**Review by ickynavigator — CHANGES_REQUESTED** [review]

**Pull request closed without merging**